<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setsourcedialog</class>
 <widget class="QDialog" name="Setsourcedialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>47</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_freq">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>frequency</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_value">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OK">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_value">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>amplitude/value</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_freq">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_phase">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>phase</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_phase">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton_DC">
      <property name="text">
       <string>DC</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_AC">
      <property name="text">
       <string>SINE</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_SQ">
      <property name="text">
       <string>SQUARE</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_value_unit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>V/A</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_freq_unit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1/s</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_value_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>290</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>rad</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
